test/contract-proxy-kit.ts

```diff
@@ -85,8 +88,67 @@ describe('CPK', () => {
     should.exist(CPK);
   });
 
-  it('should not produce CPK instances when options are missing', async () => {
-    await CPK.create(undefined as any).should.be.rejectedWith('missing options');
+  it('should produce uninitialized CPK instances when running standalone and options are missing', async () => {
+    const cpk = await CPK.create(undefined as any)
```

Comment:
Perhaps we should change the API and make the options object parameter optional.

Reply:
Sorry, I don't understand. It is optional

Reply:
I meant perhaps we should make the CPK constructor and CPK#create take `opts?` as a parameter instead of a required `opts`.

Reply:
That is how it is working right now

Reply:
I know understand why you where saying that. What needs to be updated then is line 92 with:
`const cpk = await CPK.create()` That must work now ;)